packages/typespec-python/generator/pygen/codegen/models/primitive_types.py

```diff
@@ -624,3 +624,29 @@ def instance_check_template(self) -> str:
     @property
     def serialization_type(self) -> str:
         return self.name
+
+
+class MultiPartFileType(PrimitiveType):
```

Comment:
With the new type, we don't need special logic in docstring/typing annotation for multipart properties anymore. And it is much easier to evolve in the future.